config.json

```diff
@@ -2,7 +2,8 @@
   "track_id": "gnu-apl",
   "language": "GNU APL",
   "active": false,
-  "test_pattern": "\\.tc$",
+  "test_pattern": "[.]tc$",
+  "setup_files": ["^[.]/test[.]apl$"],
```

Comment:
Hmm. Looking at this, it seems like a regex is really ugly if, as I suspect, this is mostly going to be individual filenames. Maybe a shell glob is the right thing...

Reply:
Yeah, I think you're probably right.

Want to give it a try with shell globs?

Reply:
Sure, let me see what that looks like.

Reply:
Just changed to glob syntax. See what you think.